cartographer_ros/cartographer_ros/offline_node_main.cc

```diff
@@ -162,7 +162,7 @@ void Run(const std::vector<string>& bag_filenames) {
   }
 
   for (const string& bag_filename : bag_filenames) {
-    if (sigint_triggered) {
+    if (!::ros::ok()) {
```

Comment:
Sorry, I did not realize that this is a revert. I thought that you initially added your own interrupt handler (probably missed this while doing git blame), and I thought it would be more clean to use the ROS one.